### PR TITLE
fix: add node 16 on ci

### PR DESCRIPTION
**Pull request by anshumanv**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [x] This is a **metadata update**

### For Bugs and Features; did you add new tests?

no

### Motivation / Use-Case

run tests on more node versions

### Breaking Changes
no

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node-version: [12.x]
+        node-version: [12.x, 14.x, 16.x]
```
Comment (snitin315) [inline .github/workflows/nodejs.yml:25]: I think checking lint one time is enough. 

Tough we can try enable node 16 for tests -

https://github.com/webpack/webpack-dev-server/blob/7e8692286c1cdb9b1e1483c3cf425cec849d9783/.github/workflows/nodejs.yml#L60

Comment (alexander-akait) [reply .github/workflows/nodejs.yml:25]: Yes, we should enable it for tests

Comment (anshumanv) [reply .github/workflows/nodejs.yml:25]: oh yep intention was to enable for test, updating

**Review by alexander-akait — APPROVED** [review]

**Comment by anshumanv** [comment]
some new warnings in node 16 so snapshot failing, maybe serialize warnings

File: test/helpers/test-bin.js [hunk 2]
```diff
@@ -56,6 +56,16 @@ function normalizeStderr(stderr, options = {}) {
 
   const networkIPv6 = internalIp.v6.sync();
 
+  // normalize node warnings
+  normalizedStderr = normalizedStderr.replace(
+    "(node:3658) [DEP0111] DeprecationWarning: Access to process.binding('http_parser') is deprecated.",
```
Comment (alexander-akait) [inline test/helpers/test-bin.js:61]: We should look how output the warning and open an issue

Comment (anshumanv) [reply test/helpers/test-bin.js:61]: hm let me see

Comment (anshumanv) [reply test/helpers/test-bin.js:61]: issue - https://github.com/spdy-http2/node-spdy/issues/380

Comment (anshumanv) [reply test/helpers/test-bin.js:61]: we can't migrate on `http2` because express doesn't support it on Node > 10, lets serialize warnings for now

Comment (alexander-akait) [reply test/helpers/test-bin.js:61]: hm, it can be problem... okay let fix it here

Comment (anshumanv) [reply test/helpers/test-bin.js:61]: serialize warnings or some other fix?

Comment (alexander-akait) [reply test/helpers/test-bin.js:61]: any solutions, just do it green :smile:

**Review by snitin315 — APPROVED** [review]

**Pull request merged**